### ISSUE TITLE
Valdemar Erlingsson Why Have You Abandoned This Project?

### ISSUE THREAD

**Issue by Tony-Pappas** [ValdemarOrn/SharpSoundDevice#4]
Valdemar Erlingsson Why Have You Abandoned This Project?

**Comment by ValdemarOrn** [comment]
Hi. Thanks for your interest in the project. This is a side project that I try to maintain along with my full time job. I also have several other projects on the go at any moment, with limited time to split between them. 

I see a few issue have been raised recently, I'll try and review them and hopefully I can sort them out and update dependencies and build issues where ever possible.

**Comment by ValdemarOrn** [comment]
I have just pushed a new version of the project, v1.5.0
This brings a few much-needed updates:

- Improved logging of exceptions thrown inside the wrapper
- The ability to append you own log handlers (via Logging.LogHandlers)
- Cached sample buffers for input data (previously a new buffer was allocated on each invoke of Process: not ideal from a GC perspective)
- Improved exception handling at initialization time - The framework should now provide better logs when the host can't load the plugin. Check %APPDATA%\SharpSoundDevice for the logs.
- Re-targets solution to MSVC 1.4.1 (Microsoft Visual C Runtime) - the Visual Studio 2017 version.
- Added documentation in readme.md about the MSVC

I've tested it fairly well with my existing plugins, but there's always a chance I missed something, in which case, 1.5.1 might get released very soon (but hopefully it won't be needed :)
